src/server/http/urlLookup.js

```diff
@@ -0,0 +1,99 @@
+const crypto = require('crypto');
+
+export default function (server) {
+  async function updateMetadata(urlId, urlDoc) {
+    const client = server.plugins.elasticsearch.client;
+
+    try {
```

Comment:
I think this try/catch around the async call is problematic. By the time any exception is thrown, the try/catch has finished. If you look at the transpiled code for this, you'll see that it doesn't even include the try/catch.